cmd/api/rest/config.go

```diff
@@ -0,0 +1,41 @@
+package rest
+
+import (
+	"fmt"
+	"github.com/edgecomllc/eupf/cmd/config"
+	"github.com/edgecomllc/eupf/cmd/core"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+// DisplayConfig godoc
+// @Summary Display configuration
+// @Description Display configuration
+// @Tags Configuration
+// @Produce  json
+// @Success 200 {object} config.UpfConfig
+// @Router /config [get]
+func (h *ApiHandler) displayConfig(c *gin.Context) {
```

Comment:
If I'm not mistaken, public (exported) method and functions should have first symbol capitalized.
And vice versa for nonexported functions.

Reply:
OK. Methods are used internally